SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialRPCService.cpp

```diff
@@ -225,7 +250,22 @@ TArray<SpatialRPCService::UpdateToSend> SpatialRPCService::GetRPCsAndAcksToSend(
 		SpatialRPCService::UpdateToSend& UpdateToSend = UpdatesToSend.AddZeroed_GetRef();
 		UpdateToSend.EntityId = It.Key.EntityId;
 		UpdateToSend.Update.component_id = It.Key.ComponentId;
-		UpdateToSend.Update.schema_type = It.Value;
+		UpdateToSend.Update.schema_type = It.Value.Update;
+
+		if (EventTracer != nullptr)
+		{
+			if (It.Value.SpanIds.Num() > 1)
```

Comment:
I remember seeing Jon's comment on this. It might be better to always do the merge, even when there's only 1 cause, so the chain of events looks the same.